### PR TITLE
2.4.2 whats new

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) updates the `whats-new.yml` to include PRs merged to the `2.4.2-develop` branch so that we can update the page for the GA release.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable: 2.4.2

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/whats-new.html

### CODE REVIEW

File: src/_data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,83 @@ description: This page contains recent changes that we think you'd like to know
...
+  date: February 9, 2021
+  link: https://github.com/magento/merchdocs/pull/1022
+- description: Added the [Purchase order payment methods](https://docs.magento.com/user-guide/customers/account-dashboard-my-purchase-orders.html#purchase-order-payment-methods)
+    section to include information about limitations around payments when processing
+    purchase orders.
+  versions: 2.4.2
+  type: Minor update
+  date: February 9, 2021
+  link: https://github.com/magento/merchdocs/pull/976
+- description: 'Multiple updates across the Page Builder section to reflect changes
```
Comment (erikmarr) [inline src/_data/whats-new.yml:34]:
```suggestion
- description: Added multiple updates across the Page Builder section to reflect changes
```

File: src/_data/whats-new.yml [hunk 2]
```diff
@@ -4,8 +4,83 @@ description: This page contains recent changes that we think you'd like to know
...
+    to the [Page Builder stage](https://docs.magento.com/user-guide/cms/page-builder-workspace.html#stage),
+    which is now initially blank and supports adding the following content types directly
+    to the stage: Rows, Columns, Tabs, HTML Code, Blocks, Dynamic Blocks.'
```
Comment (erikmarr) [inline src/_data/whats-new.yml:37]:
```suggestion
    to the stage: Rows, Columns, Tabs, HTML Code, Blocks, Dynamic Blocks.
```

File: src/_data/whats-new.yml [hunk 3]
```diff
@@ -4,8 +4,83 @@ description: This page contains recent changes that we think you'd like to know
...
+  type: Major update
+  date: February 9, 2021
+  link: https://github.com/magento/merchdocs/pull/938
+- description: Added a method for [importing images with remote storage](https://docs.magento.com/user-guide/system/data-import-product-images.html#method-3-import-images-with-remote-storage).
+   Added a new data transfer example for [downloadable products](https://docs.magento.com/user-guide/system/data-transfer-downloadable-products.html).
+  versions: 2.4.2
+  type: New topic
+  date: February 9, 2021
+  link: https://github.com/magento/merchdocs/pull/900
+- description: Updates to for the latest release of the dotdigital bundled extension:<br/>-
```
Comment (erikmarr) [inline src/_data/whats-new.yml:56]:
```suggestion
- description: Updates for the latest release of the dotdigital bundled extension:<br/>-
```

**Review by erikmarr — COMMENTED** [review]
Minor edits.

File: src/_data/whats-new.yml [hunk 4]
```diff
@@ -4,8 +4,83 @@ description: This page contains recent changes that we think you'd like to know
...
+    section to include information about limitations around payments when processing
+    purchase orders.
+  versions: 2.4.2
+  type: Minor update
+  date: February 9, 2021
+  link: https://github.com/magento/merchdocs/pull/976
+- description: Added multiple updates across the Page Builder section to reflect changes
+    to the [Page Builder stage](https://docs.magento.com/user-guide/cms/page-builder-workspace.html#stage),
+    which is now initially blank and supports adding the following content types directly
+    to the stage: Rows, Columns, Tabs, HTML Code, Blocks, Dynamic Blocks.
```
Comment (leslietilling) [inline src/_data/whats-new.yml:37]:
```suggestion
    to the stage for Rows, Columns, Tabs, HTML Code, Blocks, and Dynamic Blocks.
```

**Review by erikmarr — APPROVED** [review]

**Comment by leslietilling** [comment]
running tests

**Pull request merged**